drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -169,7 +163,7 @@ class << self
     hud_org_ids = tuples.map(&:second)
     raise 'orgs are in multiple data sources' if ds_ids.size > 1
 
-    joins(:projects_including_wip).where(p_t[:organization_id].in(hud_org_ids).and(p_t[:data_source_id].eq(ds_ids.first)))
+    joins(:projects).where(p_t[:organization_id].in(hud_org_ids).and(p_t[:data_source_id].eq(ds_ids.first)))
   end
```

Comment:
I'm probably missing something, but this looks like a more complicated version of:

```
joins(projects: :organization).merge(Hmis::Hud::Organization.where(id: organization_ids))
```

Reply:
yep Elliot's suggestion seems correct